QuizApp/Resources/Colors/Colors.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+enum Colors {
```

Comment:
How about we simplify this:
```
extension UIColor {
    static let loginBackground = UIColor(named: "LoginBackground")!
}
```

That way you can use it like `view.backgroundColor = .loginBackground`

Reply:
Sure, can do it that way :)